modules/retail/pages/retail-deploy-terminals-other.adoc

```diff
@@ -212,3 +212,94 @@ Then you can specify a second Salt state with the wireless details for use durin
 The boot phase supports only WPA2 PSK wireless configuration.
 Salt-managed production configuration supports all features supported by all major operating systems.
 ====
+
+
+
+== Deploy Terminals and Auto-Accept Keys
+
+You can configure {productname} to automatically accept the keys of newly deployed terminals.
+This is achieved using Salt grains.
+For more information about automatically accepting Salt grains, see https://docs.saltstack.com/en/latest/topics/tutorials/autoaccept_grains.html.
+
+[IMPORTANT]
+====
+Automatically accepting keys is less secure than manually checking and accepting keys.
+Only use this method on trusted networks.
+====
+
+
+To do this, you first need to configure Saltboot, then configure the {productname} Server.
+You can configure Saltboot to automatically sign grains once, or set it to always auto-sign.
+Alternatively, you can configure Saltboot during PXE boot using kernel parameters.
+
+
+.Procedure: Configuring Saltboot to Auto-Sign Once
+. On the branch server, create a configuration file called [path]``./etc/salt/minion.d/autosign-grains-onetime.conf``.
+. Edit the new configuration file with these details.
+  You can use any value you like as the autosign key:
++
+----
+# create the grain
+grains:
+    autosign_key: <AUTOSIGN_KEY>
+
+# send the grain as part of auth request
+autosign_grains:
+    - autosign_key
+----
+. At the command prompt, add the new configuration file to the existing ``initrd``:
++
+----
+echo ./etc/salt/minion.d/autosign-grains-onetime.conf | /
+cpio -H newc -o | gzip >> /srv/saltboot/boot/initrd.gz
+----
+
+
+
+.Procedure: Configuring Saltboot to Always Auto-Sign
+. On the Salt client, as root, create a configuration file called [path]``etc/salt/minion.d/autosign-grains.conf``.
+. Edit the new configuration file with these details.
```

Comment:
I think this is meant to be the method 1 above, so this is not "Always Auto-Sign" and it should describe modifying image sources (on build host or in git) before building.

Because the way how the images are built, the file is first included in system image and then copied to intrd, so the grains are always visible in system image - so this method works only for autosign-grains.conf, not for autosign-grains-onetime.conf

Reply:
updated step 1